lib/moon_web/pages/components/input_group_page.ex

```diff
@@ -0,0 +1,328 @@
+        </ComponentPageDescription>
+
+        <Context put={theme_class: @theme_name}>
+          <ExampleAndCode id="group_1" title="Default">
+            <:example>
+              <LeftToRight class="items-center justify-around w-full items-end">
+                <InputGroup :let={input_group_class: input_group_class}>
+                  <TextInput size="xl" placeholder="Country" class={input_group_class} />
+                  <TextInput size="xl" placeholder="Phone" class={input_group_class} />
+                </InputGroup>
+
+                <InputGroup orientation="vertical" :let={input_group_class: input_group_class}>
+                  <TextInput size="xl" placeholder="Country" class={input_group_class} />
+                  <TextInput size="xl" placeholder="Phone" class={input_group_class} />
+                </InputGroup>
+              </LeftToRight>
+            </:example>
+
+            <:code>{input_group_1_code()}</:code>
+          </ExampleAndCode>
+
+          <ExampleAndCode id="group_10" title="RTL">
+            <:example>
+              <LeftToRight class="items-center justify-around w-full items-end">
+                <InputGroup dir="rtl" :let={input_group_class: input_group_class}>
+                  <TextInput size="xl" placeholder="Country" class={input_group_class} dir="rtl" />
+                  <TextInput size="xl" placeholder="Phone" class={input_group_class} dir="rtl" />
+                </InputGroup>
+
+                <InputGroup dir="rtl" orientation="vertical" :let={input_group_class: input_group_class}>
+                  <TextInput size="xl" placeholder="Country" dir="rtl" class={input_group_class} />
+                  <TextInput size="xl" placeholder="Phone" dir="rtl" class={input_group_class} />
+                </InputGroup>
+              </LeftToRight>
+            </:example>
+
+            <:code>{input_group_10_code()}</:code>
+          </ExampleAndCode>
+
+          <ExampleAndCode id="group_100" title="Text type variants">
+            <:example>
+              <LeftToRight class="items-center justify-around w-full items-end">
+                <InputGroup :let={input_group_class: input_group_class}>
+                  <TextInput size="xl" type="email" placeholder="Email" class={input_group_class} />
+                  <TextInput
+                    size="xl"
+                    type="password"
+                    id="password1"
+                    placeholder="Password"
+                    class={input_group_class}
+                  />
+                </InputGroup>
+
+                <InputGroup orientation="vertical" :let={input_group_class: input_group_class}>
+                  <TextInput size="xl" type="email" placeholder="Email" class={input_group_class} />
+                  <TextInput
+                    size="xl"
+                    type="password"
+                    id="password2"
+                    placeholder="Password"
+                    class={input_group_class}
```

Comment:
Why TextInput receives input_group_class and sets it on itself? (Bad naming)

Why it is slot arg? It could be used from context, in TextInput.